### PR TITLE
Added a basic donation info and link

### PR DESCRIPTION
**Pull request by nickoneill**

### CODE REVIEW

**Comment by nickoneill** [comment]
This PR should include non-profit status note in the footer as well

File: static/js/components/hypothesis.js [hunk 1]
```diff
@@ -19,6 +19,9 @@ module.exports = (state, prev, send) => {
         <p>Calling is the most effective way to influence your representative. Read more about <a href="/about">why calling works.</a>
         </p>
 
+        <p>You can contribute to help out with costs associated with running 5 Calls. This project is run by an <strong>all-volunteer team</strong>, and all donations go towards basic expenses like <strong>hosting our site, sending issue alerts and keeping our data updated</strong>.</p>
```
Comment (oderby) [inline static/js/components/hypothesis.js:22]: Perhaps this donation blurb should go after the 5calls does blurb? That is, I'm suggesting moving these 2 lines to line 31.

**Review by oderby — APPROVED** [review]
Otherwise, the code looks fine to me.

File: static/js/components/hypothesis.js (hunk 1 above)
Comment (nickoneill) [reply static/js/components/hypothesis.js:22]: Erring on the side of above-the-fold (for smaller screens?) for now. There is a high likelihood that we're going to take a closer look at the homepage design soonish anyways.

**Pull request merged**